Fix Rake copyright tasks

myabc opened the pull request:
Update source file paths, ensuring copyright tasks can once again be run.

https://community.openproject.org/work_packages/18714

myabc committed:
Prune old paths from copyright tasks

[ci skip]

Signed-off-by: Alex Coles <alex@alexbcoles.com>

myabc committed:
Update copyright task exclude paths (.js)

[ci skip]

Signed-off-by: Alex Coles <alex@alexbcoles.com>

myabc committed:
Update copyright task exclude paths (.html)

[ci skip]

Signed-off-by: Alex Coles <alex@alexbcoles.com>

myabc committed:
Remove copyright task for Markdown (.md) files

[ci skip]

Signed-off-by: Alex Coles <alex@alexbcoles.com>

TeatroIO commented:
I've prepared a stage to preview changes. <a href='http://myabc-fix-rake-copyright-tasks.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://myabc-fix-rake-copyright-tasks.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

myabc committed:
Exclude application.js.erb from copyright task

This file relies on Sprockets directives, which will not be processed
properly with an ERB comment at the beginning of the file.

[ci skip]

Signed-off-by: Alex Coles <alex@alexbcoles.com>
